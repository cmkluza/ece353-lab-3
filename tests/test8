./sim-mips -b 3 3 3 ./tests/asm/test4.s ./tests/out/test8.out
./tests/sols/test4.sol