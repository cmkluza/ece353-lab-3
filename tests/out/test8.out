program name: ./tests/asm/test4.s
stage utilization: 0.960784  0.137255  0.941176  0.000000  0.196078 
register values 0  0  0  0  0  0  0  1  2  3  4  5  3  0  0  21  22  255  170  0  0  0  0  0  0  0  0  0  0  0  0  56
